commands/team_e2e_test.go

```diff
@@ -51,3 +51,55 @@ func (s *MmctlE2ETestSuite) TestRenameTeamCmdF() {
 		s.Equal("Cannot rename team '"+s.th.BasicTeam.Name+"', error : : You do not have the appropriate permissions., ", err.Error())
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestRestoreTeamsCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Restore team", func(c client.Client) {
+		printer.Clean()
+		printer.SetFormat(printer.FormatPlain)
```

Comment:
Rather than using the plain format, which is more brittle than the JSON one, would you be open to change the PR to use the default format? This way you could use the `DeleteAt` property of the returned team to check if it has been restored by the server.

I'm thinking on removing the two `printer.SetFormat` lines and doing something like this at the end of the test case:

```go
err := restoreTeamsCmdF(c, &cobra.Command{}, []string{team.Name})
s.Require().Nil(err)
s.Require().Len(printer.GetErrorLines(), 0)
s.Require().Len(printer.GetLines(), 1)
s.Require().Zero(printer.GetLines()[0].(*model.Team).DeleteAt)
```